### ISSUE TITLE
IRC Server for testing not works

### ISSUE THREAD

**Issue by jaraco** [jaraco/irc#60]
```
#!python

[nekmo@homura ~]$ python2 -m irc.server
INFO:__main__:Starting irc.server
INFO:__main__:Listening on 127.0.0.1:6667
----------------------------------------
Exception happened during processing of request from ('127.0.0.1', 37016)
Traceback (most recent call last):
  File "/usr/lib/python2.7/SocketServer.py", line 599, in process_request_thread
    self.finish_request(request, client_address)
  File "/usr/lib/python2.7/SocketServer.py", line 334, in finish_request
    self.RequestHandlerClass(request, client_address, self)
  File "build/bdist.linux-x86_64/egg/irc/server.py", line 109, in __init__
TypeError: must be type, not classobj
----------------------------------------
```

OS: Arch Linux. Package version: irc==12.1.1

The error occurs inside (clean installation) and outside (system) of a virtualenv.

Tested with Python IRC Bot (usign this library) and with Xchat.

---
- Bitbucket: https://bitbucket.org/jaraco/irc/issue/60
- Originally reported by: [Nekmo](http://bitbucket.org/Nekmo)
- Originally created at: 2015-04-25T04:15:41.926

**Comment by jaraco** [comment]
On Python 2, use old syntax to call parent **init**, as BaseRequestHandler is an old-style class. Fixes #60.

→ <<cset 8369c132dd35>>

---

Original comment by: Jason R. Coombs

**Comment by jaraco** [comment]
Released as 12.1.2. Thanks for the report.

---

Original comment by: Jason R. Coombs
